### ISSUE TITLE
Agent failing to install

### ISSUE THREAD

**Issue by TokinRing** [picluster/picluster#65]
I am getting the following error while trying to `npm install` the agent script.

> npm ERR! Command failed: /usr/bin/git clone --depth=1 -q -b master https://github.com/aidanharris/getos.git /root/.npm/_cacache/tmp/git-clone-cedb67f3
> npm ERR! /root/.npm/_cacache/tmp/git-clone-cedb67f3/.git: Permission denied
> npm ERR!

I'm running as root, so I don't understand why I am getting a permission denied error.

**Comment by rusher81572** [comment]
Worked for me on MacOS
```
 getos@3.0.1 postinstall /private/tmp/picluster/agent/node_modules/getos
> node postinstall.js

npm WARN picluster-agent@0.0.1 No description
npm WARN picluster-agent@0.0.1 No repository field.
npm WARN picluster-agent@0.0.1 No license field.

added 163 packages in 3.66s


   ╭─────────────────────────────────────╮
   │                                     │
   │   Update available 5.0.3 → 5.3.0    │
   │     Run npm i -g npm to update      │
   │                                     │
   ╰─────────────────────────────────────╯
```

**Comment by aidanharris** [comment]
@TokinRing That was fixed in 944104770600d961397e70843b796f6743db2115 and 2ea11d5d9084308e171e833b482f95ad917db6cb.
